geppetto-showcase/package.json

```diff
@@ -14,7 +14,7 @@
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-react": "^7.8.3",
-    "@geppettoengine/geppetto-client": "file:../geppetto.js",
+    "@geppettoengine/geppetto-client": "file:.yalc/@geppettoengine/geppetto-client",
```

Comment:
there is an open PR #53  for this feature, this explains why the deployment here failed, yalc steps need to be added to the Dockerfile as well